pkg/config/kube/conversion.go

```diff
@@ -32,6 +32,17 @@ import (
 	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var (
+	wellKnownPorts = map[int32]protocol.Instance{
+		25:    protocol.TCP,   // SMTP
+		53:    protocol.TCP,   // DNS. Default TCP if not specified.
+		3306:  protocol.MySQL, // MySQL
+		4222:  protocol.TCP,   // NATS
+		8086:  protocol.TCP,   // InfluxDB
```

Comment:
please don't have magic on ports as common as 8086.. port 53 is ok since that is always dns but this is not even a special port

Reply:
There is already enough (good) magic with this feature :)